web/modules/custom/dpl_related_content/src/Services/RelatedContent.php

```diff
@@ -92,10 +94,18 @@ class RelatedContent {
    */
   private array $branches = [];
 
+  /**
+   * What the results are based on - helps with debugging.
+   *
+   * @var string[]
+   *  The types of filters that can be used.
+   */
+  private array $resultBasis = [];
```

Comment:
I'll update the description.

Reply:
* The result basis is shown in the frontend, in a data-attribute as part
   * of the list. It helps developers and technical editors see what the results
   * are based on. E.g. it may say "tags, categories".
   * In the future, we may want to display this clearer for editors.